### ISSUE TITLE
Being told that transient should depend on a newer emacs or transient

### ISSUE THREAD

**Issue by tarsius** [purcell/package-lint#281]
I get a few screens full of this:

```
transient.el:L:C: error: You should depend on (emacs "28.1") or the transient package if you need `transient-something'.
```

Also note that the stand-alone `transient` isn't a "backport". It's the real thing, and Emacs bundles a, usually outdated, copy.

**Comment by tarsius** [comment]
#236 is related. I expect that for `org` one would now also get a "... or the org package" false-positive.

**Comment by purcell** [comment]
I have a branch locally where I'm working on this. 👍

**Comment by purcell** [comment]
Better late than never, I think I've resolved this.

**Comment by tarsius** [comment]
Thanks! ❤️

I can confirm it works.

Happy holidays!

**Comment by purcell** [comment]
> Happy holidays!

Same to you! 🤝
